langserver/handler.go

```diff
@@ -0,0 +1,65 @@
+package langserver
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+
+	"github.com/haya14busa/go-vimlparser"
+	"github.com/haya14busa/go-vimlparser/ast"
+
+	"github.com/sourcegraph/jsonrpc2"
+)
+
+func NewHandler() jsonrpc2.Handler {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function NewHandler should have comment or be unexported